chaos.sh

```diff
@@ -5,15 +5,31 @@ set -ex
 : ${DELAY:=30}
 : ${NAMESPACE:=default}
 
-while true; do
-  kubectl \
-    --namespace "${NAMESPACE}" \
-    -o 'jsonpath={.items[*].metadata.name}' \
-    get pods | \
-      tr " " "\n" | \
-      shuf | \
-      head -n 1 |
-      xargs -t --no-run-if-empty \
-        kubectl --namespace "${NAMESPACE}" delete pod
-  sleep "${DELAY}"
-done
+if [ "${SELECTOR_MODE}" == "equality-based" ] || [ "${SELECTOR_MODE}" == "set-based" ]; then 
```

Comment:
Suggestion: use `[ -n "${SELECTOR}" ]` here to opt into the selector behavior if a non-zero-length selector is provided.